issue.go

```diff
@@ -960,3 +969,65 @@ func (s *IssueService) Delete(issueID string) (*Response, error) {
 	resp, err := s.client.Do(req, nil)
 	return resp, err
 }
+
+// GetWatchers wil return all the users watching/observing the given issue
```

Comment:
Can you please add a link to the respective documentation here?
See the godoc of other API calls as example